posthog/models/filters/mixins/common.py

```diff
@@ -115,6 +115,10 @@ def breakdown(self) -> Optional[Union[str, List[Union[str, int]]]]:
     def breakdown_limit(self) -> Optional[int]:
         return self._data.get(BREAKDOWN_LIMIT)
 
+    @cached_property
+    def breakdown_limit_or_default(self) -> Optional[int]:
```

Comment:
Nits:
- No reason to cache this - `@property` would be more appropriate
- The type is wrong. This always returns an int

It also kind of feels wrong to have both `breakdown_limit` and this. If `breakdown_limit` has no valid usecases, perhaps underscore it?